### ISSUE TITLE
pre-configured industry solutions

### ISSUE THREAD

**Issue by AlexanderF123** [frappe/erpnext#32535]
ERPnext is a feature rich standard software that can be used for many (if not all) industries. These industries are very different,
e.g.
- hotel industry
- online shops
- producing companies
- service companies
- etc.etc.

Setting up ERPnext for the own industry is therefore not that easy, needs a lot of knowledge and experience.

I suggest that there should be a feature to save configurations for an industry for the use of other users, 

In addition there should be a store where the user can select and install those configurations. When installing ERPnext the new user should be asked about his industry so that the best configuration can be installed. With this, the start would be way easier and many new and happy users would use ERPnext.

**Comment by satisharalkar** [comment]
Yes, this will be helpful to new users and it will be easy for new site creation.
